tensorflow/tools/pip_package/setup.py

```diff
@@ -56,6 +56,7 @@
     'google_pasta >= 0.1.2',
     'keras_applications >= 1.0.6',
     'keras_preprocessing >= 1.0.5',
+    'kissfft >= 1.3.0, < 1.3.1',
```

Comment:
I'm not seeing this package in pypi, is that going to be a problem?

Reply:
Good catch - it likely will. Preparing another PR to undo some of my attempts to shove in kissfft. Thanks!